pkg/task/backup.go

```diff
@@ -60,6 +62,7 @@ func DefineBackupFlags(flags *pflag.FlagSet) {
 	flags.String(flagBackupTS, "", "the backup ts support TSO or datetime,"+
 		" e.g. '400036290571534337', '2018-05-11 01:42:23'")
 	flags.Int64(flagGCTTL, backup.DefaultBRGCSafePointTTL, "the TTL (in seconds) that PD holds for BR's GC safepoint")
+	flags.String(flagCompressionType, "", "backup sst file compression algorithm, value can be one of 'lz4|zstd|snappy'")
```

Comment:
provide a default value?

```suggestion
	flags.String(flagCompressionType, "zstd", "backup sst file compression algorithm, value can be one of 'lz4|zstd|snappy'")
```

Reply:
the default value should be default, which means the compression type will be determined by tikv.  In currently logic, tikv will choose the fastest algorithm and in most case it will be 'lz4'.

Reply:
the default is called "unknown" though

Reply:
e, so we should change the name in kv-proto to default?

Reply:
We should provide a validate default value, "zstd" looks good to me.

Reply:
Ok. Then I think we can set default to `lz4` because it's the current compression type tikv uses. So if user don't set this flag, the behavior is same with previous version, so it's compatible will old version